ajax/libs/react-popper/package.json

```diff
@@ -0,0 +1,30 @@
+
+{
+  "name": "react-popper",
+  "filename": "react-popper.min.js",
+  "description": "React wrapper around PopperJS.",
+  "keywords": [
+    "react",
+    "react-popper",
+    "popperjs",
+    "component",
+    "drop",
+    "tooltip",
+    "popover"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/souporserious/react-popper"
+  },
+  "license": "MIT",
+  "author": "Travis Arnold <travis@souporserious.com> (http://souporserious.com)",
```

Comment:
Let's remove `(http://souporserious.com)` and `<travis@souporserious.com>`
Because DNS record not found.
(And we can use `ftntravis@gmail.com` and http://travisrayarnold.com from author's info https://github.com/souporserious)
Thank you.

Reply:
Sure, but those values were directly from the package.json in the project.